config/dev/spm/sku/eg/cr/BUILD.bazel

```diff
@@ -0,0 +1,72 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+load(
+    "//rules:hsm.bzl",
+    "hsm_certgen",
+    "hsm_certgen_tar",
+    "hsm_certificate_authority_intermediate",
+    "hsm_config_tar",
+)
+load(
+    "//rules:hsmtool.bzl",
+    "HSMTOOL_CONST",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+hsm_certificate_authority_intermediate(
+    name = "cr01-ica-dice-key-p256-v0",
+    curve = HSMTOOL_CONST.ECC_CURVE.PRIME256V1,
+)
+
+hsm_certificate_authority_intermediate(
+    name = "cr01-ica-ext-key-p256-v0",
+    curve = HSMTOOL_CONST.ECC_CURVE.PRIME256V1,
+)
+
+hsm_certgen(
+    name = "cr01_ca_int_dice",
```

Comment:
to make the sival SKU consistent, should update [this](https://github.com/lowRISC/opentitan-provisioning/blob/654a9e3b2ea0142b57356146581b2da4458fbb29/config/dev/spm/sku/sival/BUILD.bazel#L64) to `sv00_int_dice`?